### PR TITLE
Fix/make api client avail

### PR DESCRIPTION
**Pull request by ChawinTan**
## Self-checks

- [x] I'm familiar with and follow this [Typescript guideline](https://basarat.gitbook.io/typescript/styleguide)
- [ ] I added unit-tests for new components
- [ ] I tried testing edge cases
- [ ] I tested the behavior of the components that interact with the backend, with an e2e test

## Discussion

Made a symlink and a script to build `api-client` so as to allow the front end to subscribe to states from the dashboard.

@codebot  and @koonpeng Do you think these changes are needed for the time being or are we sticking to `ros2-bridge` for April? If not then I will leave it as a draft request for now

### CODE REVIEW

**Comment by koonpeng** [comment]
tests are failing because of lint errors, you can ignore the rule for the generated files or change the template in https://github.com/open-rmf/ts_ros

**Comment by ChawinTan** [comment]
I'm getting a bunch of errors locally because of missing `.srv` files. Are they needed?
srv_errors

File: packages/api-server/api_server/app.py [hunk 1]
```diff
@@ -166,7 +166,8 @@ async def on_shutdown():
     logger.info("shutdown app")
 
 
-sio = socketio.AsyncServer(async_mode="asgi")
+# TODO - change cors_allowed_origin to more specific origin. Putting a wild card for now
```
Comment (koonpeng) [inline packages/api-server/api_server/app.py:169]: I think this would depend on the use case of api-server. In order for it to be consumed by any web page, we need cors *. If we only want our own pages to consume the apis, then we need a config to specify the allowed origins. In either case, apps can always consume the apis, this only affects pages in browser.

It is hard to say how * cors would impact security, we are not using cookies so that limits the scope of xsrf attacks, but of course limiting the origin would always be more secure, at the expense of blocking off third party pages (if we have any intention for third party pages to make use of our apis).

**Review by koonpeng — APPROVED** [review]

File: packages/api-server/api_server/app.py (hunk 1 above)
Comment (ChawinTan) [reply packages/api-server/api_server/app.py:169]: I see! I am still making the assumption that everything is air-gapped and there won't be any other use case anytime soon

**Review by cnboonhan — APPROVED** [review]
LTGM! :+1:  thanks for pushing this through so swiftly!

File: packages/api-server/api_server/app.py (hunk 1 above)
Comment (koonpeng) [reply packages/api-server/api_server/app.py:169]: Note that being airgap doesn't actually affect this attack vector. It reduces the chance of a malicious actor taking advantage of such vulnerability, but someone that has access to the network, but doesn't have access to certain admin functions may make use of security vulnerabililties like these to do things they should not be doing.

I don't think a * here opens any vulnerabilities, but I could be wrong, in any case, there should be a security audit to sniff out vulnerabilities.
